### PR TITLE
update helm chart for release v0.8.0

### LINKED ISSUE #2002: Update KServe helm chart for 0.8 release
/kind feature

**Describe the solution you'd like**
Update kserve [helm chart ](https://github.com/kserve/kserve/tree/master/manifests/charts) for 0.8 release

**Anything else you would like to add:**
- Add cluster serving runtime manifests
- Update values.yaml for version updates

### PR DESCRIPTION
**Pull request by Suresh-Nakkeran**
Signed-off-by: Suresh Nakkeran <suresh.n@ideas2it.com>

fixes #2002

### CODE REVIEW

File: manifests/charts/templates/configmap.yaml [hunk 1]
```diff
@@ -171,27 +157,19 @@ data:
...
-            "defaultGpuImageVersion": "{{ .Values.kserve.servingruntime.pytorch.v1.defaultGpuVersion }}",
             "supportedFrameworks": [
               "pytorch"
             ],
             "multiModelServer": false
           },
           "v2" : {
-            "image": "{{ .Values.kserve.servingruntime.pytorch.v2.image }}",
-            "defaultImageVersion": "{{ .Values.kserve.servingruntime.pytorch.v2.defaultVersion }}",
-            "defaultGpuImageVersion": "{{ .Values.kserve.servingruntime.pytorch.v2.defaultGpuVersion }}",
```
Comment (yuzisun) [inline manifests/charts/templates/configmap.yaml]: pytorch no longer need to differentiate between v1 and v2 as it is all using torchserve now, see [here](https://github.com/kserve/kserve/blob/master/config/configmap/inferenceservice.yaml#L49)

Comment (Suresh-Nakkeran) [reply manifests/charts/templates/configmap.yaml]: yes, @yuzisun .. I updated.

**Pull request merged**